fem/tmop.hpp

```diff
@@ -2118,8 +2123,19 @@ class TMOP_Integrator : public NonlinearFormIntegrator
                                        ParGridFunction &s0_hess,
                                        AdaptivityEvaluator &ahe);
 #endif
-   void GetSurfaceFittingErrors(double &err_avg, double &err_max);
-   bool IsSurfaceFittingEnabled() { return (surf_fit_gf != NULL); }
+   /** @brief Fitting of certain DOFs to given positions in physical space.
+
+       @param[in] pos     The desired positions for the mesh nodes.
+       @param[in] smarker Indicates which DOFs will be aligned.
+       @param[in] coeff   Coefficient c for the above integral. */
+   void EnableSurfaceFitting(const GridFunction &pos,
```

Comment:
Can we put the exact contribution to the TMOP objective here in terms of the inputs similar to the fitting method above?

Reply:
done